salt/cloud/clouds/scaleway.py

```diff
@@ -272,98 +272,11 @@ def __query_node_data(server_name):
         finally:
             raise SaltCloudSystemExit(str(exc))
 
-    ssh_username = config.get_cloud_config_value(
-        'ssh_username', server_, __opts__, default='root'
+    vm_['ssh_host'] = ip_address
```

Comment:
`ip_address` is not defined should be assigned from `data['public_ip']['address']`

```
if data.get('public_ip') is not None:
  ip_address = data['public_ip']['address']
```